README.md

```diff
@@ -103,16 +103,15 @@ See [Demo page][github-page]
 
 ## Props
 
-| Name                  | PropType | Description                                                                                                                   | Example                                                                                             |
-| --------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
-| googleMaps            | object   | injected by `react-google-maps-loader`                                                                                        | -                                                                                                   |
-| onNoResult            | function | Handle no results when enter key is pressed                                                                                                     | `(geocodedPrediction, originalPrediction) => {console.log(geocodedPrediction, originalPrediction)}` |
-| onSelectSuggest       | function | Handle click on suggest                                                                                                       | `(geocodedPrediction, originalPrediction) => {console.log(geocodedPrediction, originalPrediction)}` |
-| customRender          | function | Customize list item                                                                                                           | `prediction => prediction ? prediction.description : "no results"`                                  |
-| customContainerRender | function | Customize list                                                                                                                | `items => <CustomWrapper>{items.map(item => <ItemWrapper>{item.description}</ItemWrapper>)}         |
-|                       |
-| autocompletionRequest | object   | [Google map object Object](https://developers.google.com/maps/documentation/javascript/reference?hl=fr#AutocompletionRequest) | `{input: "Toulouse"}`                                                                               |
```

Comment:
Is this not being used anymore?

Reply:
I think this was just a diffing issue...